src/components/Sponsors.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Grid, Image, Header } from "semantic-ui-react";
+
+export function Sponsors() {
+  return (
+    <Grid style={{ margin: "5em 0" }} columns='equal'>
+      <Header as='h2' content='Our sponsors' />
+      <Grid.Row>
+        <Grid.Column>
+          <Image
+            src={
+              process.env.PUBLIC_URL +
+              "/images/sponsors/sponsor-gary_community_investments-300x236.png"
+            }
+          />
+        </Grid.Column>
+        <Grid.Column>
+          <Image
+            src={process.env.PUBLIC_URL + "/images/sponsors/galvanize.svg"}
+          />
+        </Grid.Column>
+        <Grid.Column>
+          <Image
+            src={
+              process.env.PUBLIC_URL +
```

Comment:
We'll need to come back and add a `.env` file to the project for this and have webpack inject the env vars at build time. is that what you were thinking ?

Reply:
Depending on the host this should just be the path to the public folder.  The other way would be to move the images under the src folder and import them directly, or just host them on contentful.  I'm not sure the best way to handle images in React.